### PR TITLE
use $Logger in cronjobs

### PR DESCRIPTION
**Pull request by rjbs**
Now that the cron jobs are all(?) successfully loading PAUSE::Logger, we should use it.  This replaces "warn" calls with use of $Logger.

### CODE REVIEW

File: cron/cron-daily.pl [hunk 1]
```diff
@@ -80,7 +80,7 @@ sub read_errorlog {
     close LOG;
     report "\nErrorlog contains $errorlines lines today\n";
   } else {
-    warn "error opening $PAUSE::Config->{HTTP_ERRORLOG}: $!";
+    $Logger->log("error opening $PAUSE::Config->{HTTP_ERRORLOG}: $!");
```
Comment (rspier) [inline cron/cron-daily.pl:83]: if this is an error, should it be fatal?

File: cron/cron-daily.pl [hunk 2]
```diff
@@ -241,7 +241,8 @@ sub watch_files {
           quoteHighBit => 1,
         );
         my $v = $d->stringify($File::Find::name);
-        warn sprintf qq{Found a bad directory v[%s], rmtree-ing}, $v;
+
```
Comment (rspier) [inline cron/cron-daily.pl:244]: unnecessary blank line?

**Review by rspier — APPROVED** [review]
github's diff is not doing a great line displaying this change.

File: cron/cron-daily.pl (hunk 1 above)
Comment (rjbs) [reply cron/cron-daily.pl:83]: I think the whole premise of this code is kinda wonky, so I'm gonna not touch it beyond changing its logging.  "Send me the day's web error logs" is odd.

File: cron/cron-daily.pl (hunk 2 above)
Comment (rjbs) [reply cron/cron-daily.pl:244]: Most blanks are, right? :)

Anyway, so much of the PAUSE code is without any vertical whitespace.  Sometimes I add some to help break things up.  This is maybe not the most obvious place to put it, but: I will continue to add vertical whitespace to create paragraphs of code.

Comment (rspier) [reply cron/cron-daily.pl:244]: I like vertical whitespace, this just felt like a weird place to put it.

**Pull request merged**